build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -539,7 +539,10 @@ message Command {
 
   // The arguments to the command. The first argument must be the path to the
   // executable, which must be either a relative path, in which case it is
-  // evaluated with respect to the input root, or an absolute path.
+  // evaluated with respect to the working directory, or an absolute path. Path
```

Comment:
The first sentence says that the first argument must be either relative or absolute, but doesn't mention the third possibility- a command name without any path separators.